### PR TITLE
DU-23/OnBoarding & Tutorial pages

### PR DESCRIPTION
**Pull request by nrios14**
Updated, restructured, and renamed the tutorials page to OnBoarding Page

Created a new Tutorials sidebar section with a Tutorials Overview page and 5 new tutorials

### CODE REVIEW

**Review by stan-travis — CHANGES_REQUESTED** [review]
Please update all snippets with file title

**Comment by nrios14** [comment]
Updates code snippets 
Renamed tutorials.md to onboarding.md 
and added re direct links

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

File: user/onboarding.md
Comment (vhadzh01) [inline user/onboarding.md:1]: Plan changed to 14 days instead of 30 days

**Review by vhadzh01 — APPROVED** [review]

**Review by vhadzh01 — APPROVED** [review]

**Review by stan-travis — APPROVED** [review]
LGTM

**Comment by vhadzh01** [comment]
Merging Tickets: DU-23 (US2) and DU-25 (US5)

**Pull request merged**